kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/data/IcebergWriter.java

```diff
@@ -77,8 +78,47 @@ public void write(SinkRecord record) {
   }
 
   private Record convertToRow(SinkRecord record) {
-    // FIXME: update this when the record converter is added
-    return null;
+    if (!config.evolveSchemaEnabled()) {
+      return recordConverter.convert(record.value());
```

Comment:
It seems we only really sink the `value` portion of the records. 
Is the idea that if users have valuable data in the `key`, they should combine this sink-connector with an appropriate SMT to move the data from the `key` into the `value`?

Reply:
Yes, that's right. The thought is we will have a subproject for useful SMTs and this might be one we would want to add.